src/content/guide/Problemsolving.md

```diff
@@ -18,27 +18,24 @@ heroImage: "../../img/guidebg-3.webp"
 
-<div class="error">Wii Update <span class="badge bg-warning">Food Channel/Photo Prints Channel</span><h4>If the system asks you to update. It means that you didn't install the "WiiLink SPD (Wii).wad" or the "WiiLink SPD (vWii).wad" (Depending on your Console) using yawmME, go to the location where you put the WiiLink SPD.wad, then install it by pressing the A Button twice on your Wii remote.</h4><hr></div>
+<div class="error">Error Code: -1017 <span class="badge bg-warning">Food Channel</span><h4>This is a known issue with the patcher! In the meantime, you will find a file titled "Food Channel (English).wad" on the root of your SD card. Use your favorite WAD manager to install!</h4><hr></div>
 
-<div class="error">Error: -1022/Error -1029 <span class="badge bg-warning">Any Channel</span><h4>Using your favorite WAD manager, Uninstall the Channel, and Install again. For many, this means going to the Location the Channel WAD, and Pressing the - (minus) button on your Wii remote, and Pressing A to Uninstall. Then, Install the WAD again.</h4><hr></div>
+<div class="error">Wii Update <span class="badge bg-warning">Food Channel/Photo Prints Channel</span><h4>If the system asks you to update. It means that you didn't install the "WiiLink Address Settings.wad". Using yawmME, go to the location where you put the "WiiLink Address Settings.wad", then install it by pressing the A Button twice on your Wii remote.</h4><hr></div>
 
-<div class="error">Pink Bar <span class="badge bg-primary">European Wiis</span><h4>If you see a Pink Bar at the bottom of your TV/Screen, Go to the Wii Settings -> Screen -> TV Resolution, then set the TV Resolution to 60 HZ (480p/480i).</h4><hr></div>
+<div class="error">Error Code: -1022/-1029 <span class="badge bg-warning">Any Channel</span><h4>Using your favorite WAD manager, uninstall the channel, and install again. For yawmME, this means locating the channel WAD, and pressing the - (minus) button on your Wii remote, and pressing A to uninstall. Then, install the WAD again.</h4><hr></div>
 
-<div class="error">Adding Food to Basket <span class="badge bg-warning">Food Channel</span><h4>Please try erasing save data for the Demae Channel, and setting it up again! Let us know in <a href="https://discord.gg/wiilink" class="text-info"><i class="fa-solid fa-message" style="margin-right:5px;"></i>#support</a> if you continue to experience issues.</h4><hr></div>
+<div class="error">Adding Food to Basket <span class="badge bg-warning">Food Channel</span><h4>Please ensure your console ID is registered as per the instructions <a href="http://localhost:4321/guide/demae/#section-ii---registering-your-console-id">here</a>, then try erasing save data for the Demae Channel, and setting it up again! Let us know in <a href="https://discord.gg/wiilink" class="text-info"><i class="fa-solid fa-message" style="margin-right:5px;"></i>#support-text</a> if you continue to experience issues.</h4><hr></div>
```

Comment:
can you replace http://localhost:4321/guide/demae/#section-ii---registering-your-console-id with https://wiilink24.com/guide/demae/#section-ii---registering-your-console-id

Reply:
thanks for pointing that out!